### PR TITLE
Fix MacOS Mojave User Perivilaged Debugging Problem 

### PR DESCRIPTION
**Pull request by SajjadPourali**
In macOS Mojave adding "com.apple.security.cs.debugger" key in entitlements is necessary for singing process, Otherwise macOS mojave will prevent attach debugger to apps in non-superuser privileges.
I also add "com.apple.security.cs.allow-dyld-environment-variables" key, Because I believe that can be useful.

REF : https://developer.apple.com/documentation/security/com_apple_security_cs_debugger
REF : https://devstreaming-cdn.apple.com/videos/wwdc/2018/702zi9t7twhu9310kz5/702/702_your_apps_and_the_future_of_macos_security.pdf?dl=1

### CODE REVIEW

File: binr/radare2/radare2_macos.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+ <dict>
+  <key>com.apple.security.cs.debugger</key>
+  <true/>
+
```
Comment (radare) [inline binr/radare2/radare2_macos.xml]: unnecessary empty line

**Comment by radare** [comment]
conflicts?

**Comment by radare** [comment]
Please rebase and squash your changes. Thanks!

**Comment by SajjadPourali** [comment]
@radare : I did fix it .

**Pull request merged**